hyrule_field.logic

```diff
@@ -1,11 +1,63 @@
 area HyruleField:
-  room EastHillsNorth
-  room EastHillsMid
-  room EastHillsSouth
+  room EastHillsNorth:
+    node West:
+      door West
+      ennemy WestAcroBandits
+      door South
+    node WestFusionChest:
+      chest WestFusionChest
+    node GoldenRopeFusion:
+      ennemy GoldenRope
+    node MainEast:
+      door FarmersHouse
+      door SouthEast
+      ennemy EastAcroBandits
+      ennemy Octorocs
+    node WoodsFairyLedge:
+      door NorthEast
+    node FarmersDiggingCave:
+      door DiggingCave
+    West -> WestFusionChest : flag WestFusionChest
```

Comment:
Do we want to have a convention for fusion names? E.g. ending them all in `Fusion` like
```suggestion
    West -> WestFusionChest : flag ChestFusion
```